git_cookbook.md

```diff
@@ -1,6 +1,22 @@
 # Git cookbook
 Handy recipes for git'n it done.
 
+
+#### Bash Completions
+Get tab-completion for all your git commands, including branch names!
+
+1. [Install homebrew](https://github.com/Homebrew/homebrew/wiki/Installation), if you haven't already.
+1. Install git and bash-completion via homebrew _(you must install git via homebrew for this to work)_
```

Comment:
If you know how to do that, I welcome your edits. I've never done it so I
don't know how.
On Apr 15, 2016 5:10 PM, "Peter Farlow" <notifications@github.com> wrote:

> In git_cookbook.md
> <https://github.com/IntrepidPursuits/sherpa/pull/36#discussion_r59941411>:
>
> > @@ -1,6 +1,22 @@
> >  # Git cookbook
> >  Handy recipes for git'n it done.
> >
> > +
> > +#### Bash Completions
> > +Get tab-completion for all your git commands, including branch names!
> > +
> > +1. [Install homebrew](https://github.com/Homebrew/homebrew/wiki/Installation), if you haven't already.
> > +1. Install git and bash-completion via homebrew _(you must install git via homebrew for this to work)_
>
> Do we want to add steps for installation if you have installed git prior
> to homebrew?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/IntrepidPursuits/sherpa/pull/36/files/192fbc191ed134ebed05d1806a920a225e1cd66c#r59941411>
>